.github/workflows/benchmark_asterinas.yml

```diff
@@ -60,15 +60,19 @@ jobs:
           - fio/ext2_seq_write_bw_no_iommu
           - fio/ext2_seq_read_bw_no_iommu
           # Network-related benchmarks
+          - lmbench/tcp_loopback_bw_128
```

Comment:
I think these benchmarks are enough, also, could you please reorder these benchmarks? I think the number of benchmarks is sufficient for us to classify further, for example:

```text
          # Loopback-related network benchmarks
          - lmbench/tcp_loopback_select_lat
          - lmbench/tcp_loopback_http_bw
          - lmbench/tcp_loopback_bw_128
          - lmbench/tcp_loopback_bw_4k
          - lmbench/tcp_loopback_bw_64k
          - lmbench/tcp_loopback_connect_lat
          - lmbench/tcp_loopback_lat
          - lmbench/udp_loopback_lat
          # VirtIO-net-related network benchmarks
          - lmbench/tcp_virtio_bw_128
          - lmbench/tcp_virtio_bw_64k
          - lmbench/tcp_virtio_connect_lat
          - lmbench/tcp_virtio_lat
          - lmbench/udp_virtio_lat 
```